distributions/docker/solr/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.7'
+services:
+  replication-solr:
+    image: cnxta/replication-solr:0.3.0-SNAPSHOT
+    ports:
+      - "8983:8983"
+    volumes:
+      - replication-data:/var/solr
+    networks:
+      - replication
+    entrypoint:
```

Comment:
:pencil2: you don't need to override the entrypoint if its got the same stuff in it :smile:

Reply:
Removed